### PR TITLE
Job Type placeholder

### PR DESCRIPTION
**Pull request by turtlepod**
Issue: https://github.com/Automattic/WP-Job-Manager/issues/794

Note: I didn't update the pot file. Please add it after merge.
(not sure why but grunt makepot did not work in my install)

### CODE REVIEW

File: includes/forms/class-wp-job-manager-form-submit-job.php [hunk 1]
```diff
@@ -173,7 +173,7 @@ public function init_fields() {
 					'label'       => __( 'Job type', 'wp-job-manager' ),
 					'type'        => $job_type,
 					'required'    => true,
-					'placeholder' => '',
+					'placeholder' => __( 'Choose job type&hellip;', 'wp-job-manager' ),
```
Comment (dbtlr) [inline includes/forms/class-wp-job-manager-form-submit-job.php]: Adding the HTML entity `&hellip;` into the translation might make translations more difficult, as translators need to know what this means AND it is inconsistent, as no other placeholders in this form use them.

I would suggest ditching the `&hellip;` but at bare minimum, move them outside of the translation like so: `__( 'Choose job type', 'wp-job-manager' ) . '&hellip;'`

**Review by dbtlr — CHANGES_REQUESTED** [review]

File: includes/forms/class-wp-job-manager-form-submit-job.php (hunk 1 above)
Comment (turtlepod) [inline includes/forms/class-wp-job-manager-form-submit-job.php]: That hellip is to override this one:
https://github.com/Automattic/WP-Job-Manager/blob/v1.14.0/wp-job-manager-functions.php#L442

Which also using hellip (so, it's consistent)
Do you want me to change it to something else too?

Comment (turtlepod) [inline includes/forms/class-wp-job-manager-form-submit-job.php]: Plus, you need to translate hellip. Your method might not work for non-latin language.
It's how WP do it too.
https://github.com/WordPress/WordPress/blob/7ad554f955a1eac48c0361c9997af5cbe8c49dd8/wp-content/themes/twentyseventeen/searchform.php#L19

Comment (dbtlr) [inline includes/forms/class-wp-job-manager-form-submit-job.php]: Fair enough :)

**Review by dbtlr — APPROVED** [review]

File: includes/forms/class-wp-job-manager-form-submit-job.php (hunk 1 above)
Comment (turtlepod) [inline includes/forms/class-wp-job-manager-form-submit-job.php]: Just info: more hellip in WP core:
https://github.com/WordPress/WordPress/search?utf8=%E2%9C%93&q=%26hellip&type=

**Comment by dbtlr** [comment]
LGTM: Still needs updated POT file though. Will merge this in later.

**Comment by turtlepod** [comment]
Yeah, I cannot make the "makepot" works in my local dev.
I'm not sure why.

Please update the pot as needed/after merge.

**Comment by turtlepod** [comment]
makepot is finally working in my local dev install. YEY!
(Sorry for the delay)

**Pull request merged**